### ISSUE TITLE
Which ARIA attributes apply to descendant elements?

### ISSUE THREAD

**Issue by JAWS-test** [w3c/aria#1063]
I suspect that not all ARIA attributes are automatically inherited by all descendant elements and should be output by AT for these descendant elements (e.g. according to the specification `aria-disabled` is inherited by descendant elements, but `aria-haspopup` probably not). I suggest:
* if all attributes are inherited to all descendant elements, this should be mentioned in the introduction
* if only certain attributes are inherited by all or certain descendant elements, this should be mentioned with the respective attribute.

The question arose because in [APG: Data Grid Examples](https://w3c.github.io/aria-practices/examples/grid/dataGrids.html) the aria-sort is not perceptible in the child element (https://github.com/w3c/aria-practices/issues/1175, https://github.com/FreedomScientific/VFO-standards-support/issues/319).

**Comment by stes-acc** [comment]
This causes much trouble and should be addressed with priority.
